tests/Filesystem/FilesystemAdapterTest.php

```diff
@@ -111,16 +111,16 @@ public function testDownloadNonAsciiFilename()
     {
         $this->filesystem->write('file.txt', 'Hello World');
         $files = new FilesystemAdapter($this->filesystem, $this->adapter);
-        $response = $files->download('file.txt', 'пиздюк.txt');
+        $response = $files->download('file.txt', 'привет.txt');
         $this->assertInstanceOf(StreamedResponse::class, $response);
         $this->assertSame("attachment; filename=pizdiuk.txt; filename*=utf-8''%D0%BF%D0%B8%D0%B7%D0%B4%D1%8E%D0%BA.txt", $response->headers->get('content-disposition'));
```

Comment:
Shouldn't this be changed as well?

Reply:
Yeah, the txt name in both assertSame methods should be updated.

Reply:
Thanks, fixed!